extensions/amp-reach-player/amp-reach-player.md

```diff
@@ -20,13 +20,12 @@ limitations under the License.
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>
-    An <code>amp-reach-player</code> component displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
+    <td>Displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
     </td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Beta</td>
+    <td>Stable</td>
```

Comment:
Okay.  I'll mark that as a future improvement.  For now, all elements have the availability "field".  I think too that I would like to use this in my future way of tagging - using meta of some sort.